index.js

```diff
@@ -1,5 +1,5 @@
-import Bluebird from "bluebird"
 import newrelic from "artsy-newrelic"
+import Bluebird from "bluebird"
```

Comment:
One of the newrelic logs is specifically that newrelic needs to load before other modules, I don't know if this is a case of "we have to do this" - so I'm highlighting this